content/spin/serverless-ai-tutorial.md

```diff
@@ -27,7 +30,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/serverless-ai
 - [Conclusion](#conclusion)
 - [Next Steps](#next-steps)
 
-AI Inferencing performs well on GPUs. However, GPU infrastructure is both scarce and expensive. This tutorial will show you how to use Fermyon Serverless AI to quickly build advanced AI-enabled serverless applications that can run on Fermyon Cloud. Your applications will benefit from 50 millisecond cold start times and operate 100x faster than other on-demand AI infrastructure services. Take a quick look at the video below, and make sure you sign up here to be one of the first to access the Fermyon [Serverless AI private beta](https://developer.fermyon.com/cloud/serverless-ai).
+Artificial Intelligence (AI) Inferencing performs well on GPUs. However, GPU infrastructure is both scarce and expensive. This tutorial will show you how to use Fermyon Serverless AI to quickly build advanced AI-enabled serverless applications that can run on Fermyon Cloud. Your applications will benefit from 50 millisecond cold start times and operate 100x faster than other on-demand AI infrastructure services. Take a quick look at the video below, and make sure you sign up here to be one of the first to access the Fermyon [Serverless AI private beta](https://developer.fermyon.com/cloud/serverless-ai).
 
 <iframe width="854" height="480" src="https://www.youtube.com/embed/01oOh3D9cVQ?si=wORKmuOkeFMGYBsQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
this seems to actually show the iframe code. was the purpose to embed a video? 
![Uploading Screenshot 2023-09-13 at 15.26.27.png…]()

Reply:
Yes, intended to embed the YouTube Video.

![Screenshot 2023-09-14 at 07 02 53](https://github.com/fermyon/developer/assets/9831342/00819f72-851f-4e40-a5af-c7929d282eb5)